### ISSUE TITLE
Clarify how Somatic and Verbal component could be used with the system

### ISSUE THREAD

**Issue by mlenser** [mlenser/kryx-rpg-issues#186]

**Comment by Marcloure** [comment]
Just to say how I do at my table, until now I have completely ignored verbal and somatic components unless the spell explicitly requires them, such as Command, Suggestion, and touch spells. If the caster wants to cast underwater, sure. 

Now, it feels strange that the caster can cast even when tied, but this situation never came up in my game, so I haven't had to worry about it yet.

**Comment by itamarcu** [comment]
This is an important issue for spellcasters under the effects of Silence or Gag. These effects are expected to at least partially neutralize spellcasters.

**Comment by Lamorak11** [comment]
I'll also add that this is an important factor for spells that might be used in a social context - whether or not your Charm Person spell has obvious verbal or somatic components is going to effect the choice to cast it in certain situations - a mage suddenly chanting and waving their hands about is a big no-no in many situations. If such spells do have said components, should there be a codified way to hide said components from the senses of others?

I think that this should be kept separate from verbal and somatic actions required specifically by a spell, such as in spells like command, and that you shouldn't be allowed to hide said actions either.

**Comment by eunomicZenith** [comment]
There's also the big question of how recognizing spells works if spells are without any visible components.

In general, I'm deeply unsatisfied with "treat them as if they had components", because they _don't_, by the current rules. By the current rules you're entirely capable of casting Charm Person completely still and silent in the middle of a ballroom, or casting spells while your hands are tied or you're gagged. Having the DM decide whether you can or cannot on a case-by-case basis ""works"" but it does so by completely sidestepping/ignoring the fundamental fact that spells are components-less in the current system as-written, and the logical discrepancies and gameplay issues this causes. What do I do if my DM rules that I can't teleport outside of a tentacle monster's grip because apparently teleport requires hand gestures (while Misty Step in 5e only required somatic casting)? I don't think the solution is "have a hard, stern talk with your DM". I believe we need written-in rules. I believe we DO need somatic and verbal components listed on each spell, because the alternative is a mess of inconsistent ad-hoc rulings that create ambiguity, conflicts and arguments.

**Comment by itamarcu** [comment]
I'm inclined to houserule it in my own games like this:

> When you cast a spell, you must either speak aloud arcane words, or a free hand (or one holding an arcane focus) to make precise gestures. Casting a spell is audible or visible this way, and creatures who see you are aware that you are casting the spell. If the spell has a casting time of one action or longer, you must use both gestures and words. The GM may rule that you cannot use your hands to cast spells, such as if your arms are restrained by vines.

It's kind of ugly, but the intention is to...
* Let Gag and Silence prevent people from casting at least some spells, and the same for things that restrain casters
* Let half-casters (who are holding weapons) cast their bonus-action attack buffs while holding a weapon
* Explain why casters aren't all holding shields all the time (and generally, make them think a little bit more about what they're holding)

**Comment by mlenser** [comment]
> Having the DM decide whether you can or cannot on a case-by-case basis ""works"" but it does so by completely sidestepping/ignoring the fundamental fact that spells are components-less in the current system as-written

They are component-less to let DMs decide. Me, personally, would expect most spells that aren't cast by a Psion to have words/gestures. I'd handle those cases when they came up (pretty much never).

I'm failing to find the blog that decries spell components from years ago.

> I'm inclined to houserule it in my own games

Imo this shouldn't apply to the Psion as they use their mind, not arcane/divine words or other things.

**Comment by Marcloure** [comment]
We need to be careful about sword+shield, since this is intended by the rules. Also, if we make some generic rule like "you need to be able to move your hands" that would limit teleportation spells for instance, that normally allow you to escape such bounds. 

Now, I do think a precise rule would be better than "let the DM rule". Currently I follow the system by its rule, so if someone wants to cast charm in the middle of a ballroom, I allow it since it doesn't require anything.

**Comment by eunomicZenith** [comment]
If it were up to me, I'd say that holding objects in your hand does not interfere with somatic components, only your arms/hands being restrained does. Let's be real, if you were a caster who wanted to hold weaponry/shields in your hands you were forced to pick the War Caster feat, by RAW, in 5e, and a lot of the time people completely ignored this and pretty much acted as if everyone had it.

I would add verbal/somatic components to spells as needed, and give the Psion a class feature that essentially lets them ignore them.

**Comment by itamarcu** [comment]
Agreed, Psions should be a special case, I'll add it to my comment.

> if we make some generic rule like "you need to be able to move your hands" that would limit teleportation spells for instance, that normally allow you to escape such bonds.

that's why I'd allow using just your voice for bonus actions spells (such as Misty Step)!

> I would add verbal/somatic components to spells as needed

I like the simplicity of the current system, where spells aren't polluted with the components, that are irrelevant in most cases.

**Comment by eunomicZenith** [comment]
> that's why I'd allow using just your voice for bonus actions spells (such as Misty Step)!
This feels very much like a hackaround, and there _have_ to be cases where this doesn't hold water, some non-bonus-action spell that we don't want to have somatic components.

**Comment by itamarcu** [comment]
....yeah, maybe. if anyone can give me some examples of spells that should be allowed like this I'll update my hacky suggestion :)

**Comment by mlenser** [comment]
I found the blog that inspired me to remove components: http://dmsworkshop.com/2017/04/14/fixing-spell-components-keep-your-flavour-mike-mearls-i-dont-like-it

___________

Looks like I need to go through every spell and add components... :worried:

@Marcloure please do it for me.

**Comment by eunomicZenith** [comment]
Here is what I suggest, summarized:
- No individual material components (bat guano, etc.), except for expensive ones. (For spell foci, see the end of this message.)
- Add verbal and somatic components, following these guidelines:
> - If we want to be able to cast it while grappled or tied up, no somatic.
> - If we want to be able to cast it while unable to speak, no verbal.
> - If we don't want to be able to cast the spell in the presence of others without exterior signs, either - or both of verbal and somatic.
- Psions get a class feature that lets them cast spells ignoring components entirely.
- A feat will be available to pick up for other spellcasters who want to ignore components (saying it's something you can learn to do, like wordless or wandless spellcasting in the Harry Potter series). This can either be the Metamagic feat (as Metamagic had 'still' and 'silent' options in 5e) or a brand new feat.

Here's my thoughts on spell foci:
Spell foci create issues old as time - and just as trivial. If a half-caster wants to cast a spell in 5e that requires a material component, despite having the War Mage feature, by RAW, they're supposed to drop something to go grab their spell focus, because the War Mage feat does not actually let them ignore that. Here's my suggestion:
- Take away the need for spell foci entirely. Spellcasters can cast spells without wands or staves (though they might be incentivized to still hold one if it's a magical item with its own benefits).
If you're not okay with that, here's an alternative.
- Spell foci can be weapons or other objects, and they don't need to be held, they can be on your person. Your spell focus can be a medallion on your chest, or a ring, or a belt, or the sword you're holding, or your shield, and so on. Now, this does create an issue if your players are munchkin-minded, because this allows them to do stuff like have their spell focus be stuff like a toe ring safely hidden away inside their shoe, which defeats the flavor and makes it pretty much impossible to take it away from them unless you strip and search them.

**Comment by itamarcu** [comment]
I think we should first consider what we want to allow, before going all in with any solutions.

* Do we want to allow a magus holding a sword and a shield to cast Fireball?

* What about two swords?

* What about a bow, or a crossbow?

* Should the rules be different for full caster sand half casters?

* How strong do we want silencing effects to be?

* What "restraining" effects do we want to have (e.g. the kind you'd use Misty Step to teleport out of)?

* When do we want spells to be visible enough to be counterspell?

* When should people be allowed to subtly cast a spell, in a way that avoids both countering it and noticing it's happening (social situations)?

**Comment by itamarcu** [comment]
Personally, I would say:

* Magi should be able to cast minor spells even if they have weapons with both hands. However, stronger spells (action spells? high-mana spells?) should be more difficult to cast with weapons in your hands.

* Keeping a hand free, or with an arcane focus, should have some benefit, even if a marginal one.

* Rules shouldn't be different for different casters, but that's a preference. Psions should still get the benefit of no components for example.

* Silence, restraints, disarming a focus, etc - should all be at least somewhat useful. In combination, they should disable most casters completely.

* Spells should always be noticeable, unless the caster makes special effort to hide them (stealth check, a feat, extra resources, illusions, etc) or if the caster is very strong (e.g. if it costs less than half the mana limit for example?)

**Comment by Marcloure** [comment]
My take:
 **Do we want to allow a magus holding a sword and a shield to cast Fireball?**
Yes. Even if he's clad in plate.

 **What about two swords?**
Yes.

 **What about a bow, or a crossbow?**
Also yes.

 **Should the rules be different for full caster sand half casters?**
No.

 **How strong do we want silencing effects to be?**
Not sure. I think I'm ok if it blocks spells, but we also have other antimagic spells for that. 

 **What "restraining" effects do we want to have (e.g. the kind you'd use Misty Step to teleport out of)?**
None? I mean, not the restrained condition at least. Other tighter restraint I think could impede spellcaster.

 **Should disarming a spellcaster of their arcane focus be a viable strategy? Should this also work against half-casters? how does this work with spellcasters whose focus isn't held?**
No. Arcane focus should not exist as a requirement. People should be able to cast spells from their hands without wands.

 **When do we want spells to be visible enough to be counterspell?**
Anytime a spell is cast and the creature sees the caster.

 **When should people be allowed to subtly cast a spell, in a way that avoids both countering it and noticing it's happening (social situations)?**
Currently, I allow always. Otherwise, I think social spells are made to be used subtly? I mean, what is the use of Friends and Judge and illusions if others know you are casting a spell and judging them?

**Comment by eunomicZenith** [comment]
I do think a spellcaster to be able to cast spells while holding weapons or other items, and that the rules shouldn't be different for half and full casters.

I don't think keeping a hand free should be required nor really provide a benefit.

If you, the DM, really do want your players to adhere to flavor and hold a focus, you can give them a magic staff (or magical sword!) and tell them that they can cast fireball once a day as long as they hold it, or that it increases their spell DC by 1, and, in 99% of cases, the end result will be that they will go around holding it.

Note that, by RAW in 5e, focuses need to be held. Again, allowing foci to simply be on your person without being held leads to players being able to use toe rings as foci, which entirely defeats the flavor, makes the focus non-visible, and makes it essentially impossible to deprive them of their focus. I'd rather just not require foci, because at that point you might as well.

I would say a majority of spells should require at least one of somatic or verbal, and as a result silencing+restraining a caster should take away the majority of their spellcasting - but not all. (Do note, I'm against foci, so it should be a given that I'm taking 'disarm them of their component' out of the equation.)
If you think you can keep a wizard hostage by simply gagging them and tying them up, you _should_ be unpleasantly surprised when that wizard gives you the middle finger and tells you "screw you, I'm magic", having found a components-less way of getting out of that predicament.

A majority of spells should have at least one of somatic and verbal, and thus a majority of spells should be visible when cast and thus counterspellable.

I do think some spells that have minor social effects (IE, yes to Friends, no to Charm Person) should have no components, and same would go for illusion spells, as being visible would defeat the point, as they are indeed meant to be subtle.

**Comment by mlenser** [comment]
I agree with all of what Marcloure wrote.

> If the caster wants to hide that, they can try with a skill check.

Stealth.


Some notes:
- Psionicists should never have to use any components for psionic power source spells. If they cast burning hands they should still have to use their hands
- Suffused should never need material components and likely other components. They cast these spell innately, not through study or memorization or some other method
- Focuses only really make sense on Acolyte (holy symbol) and Mage. Even then it's too restrictive, imo.
- Teleports shouldn't really be stopped, probablt ever.

**Comment by eunomicZenith** [comment]
> Suffused should never need material components and likely other components. They cast these spell innately, not through study or memorization or some other method.

If a wizard needs to say formulas and make hand gestures to cast given spells, so should a Suffused, IMO. Like, that's the point of the short story written in the article you linked!

**Comment by mlenser** [comment]
> If a wizard needs to say formulas and make hand gestures to cast given spells, so should a Suffused, IMO. Like, that's the point of the short story written in the article you linked!

That would necessitate the same for the psionicist. I very much disagree. These 2 casters are unique.

**Comment by eunomicZenith** [comment]
> That would necessitate the same for the psionicist.

I don't agree with this. I'd have psionicists be the one exception.

**Comment by mlenser** [comment]
> I don't agree with this.

This has been the case for many editions. Eschew Materials should inherently be part of the Suffused. Suffused are innate casters.

You're free to disagree and make your own rules, but that's how it's going to be in the official rules.

**Comment by itamarcu** [comment]
Suffused should be able to use magic with no arcane foci freely - shooting elemental blasts from your hands feels suffusedy enough for me. Maybe I'd allow them to cast cantrips even if they're gagged and tied up, but not mana magic.

**Comment by mlenser** [comment]
> Suffused should be able to use magic with no arcane foci freely - shooting elemental blasts from your hands feels suffusedy enough for me. Maybe I'd allow them to cast cantrips even if they're gagged and tied up, but not mana magic. Psionic power should feel more special, and in a world where psionicists can kill you if they're bound and gagged, people will always kill them and never keep them captive (without magical shackles or something), which has worldbuilding implications.

100%

**Comment by eunomicZenith** [comment]
> Suffused should be able to use magic with no arcane foci freely - shooting elemental blasts from your hands feels suffusedy enough for me. 

As mentioned, I'd remove the need for arcane foci entirely, from all spellcasters.

> Maybe I'd allow them to cast cantrips even if they're gagged and tied up, but not mana magic.

> This has been the case for many editions. Eschew Materials should inherently be part of the Suffused. Suffused are innate casters. Meaning no Material components or focuses.

As a note, we're talking about verbal and somatic components. I agree suffused shouldn't need foci or components. I do think they should need to make gestures and speak out their spells, when required.
Itamarcu seems to mostly agree with me on this. The article you yourself linked points out the absurdity of the way things are in 5e on this issue, and I'm surprised you seem not to agree with it despite being the one who linked it.

**Comment by itamarcu** [comment]
I think we're all in agreement, about suffused, as @mlenser said
> Suffused are innate casters. Meaning no Material components or focuses.

**Comment by eunomicZenith** [comment]
> I think we're all in agreement, about suffused, as @mlenser said
Suffused are innate casters. Meaning no Material components or focuses.

Again, as a note, I'm in favor of removing material components and foci entirely. That message, however, says nothing of verbal and somatic components.

**Comment by mlenser** [comment]
A Suffused would need to provide verbal and somatic components, but they would be different than a Mage's verbal and somatic components. A Mage would study and know the exact words and gestures. A Suffused would've just figured something out that works. An Occultist would speak strange words compared to a mage as well.

**Comment by eunomicZenith** [comment]
> A Suffused would need to provide verbal and somatic components, but they would be different than a Mage's verbal and somatic components. A Mage would study and know the exact words and gestures. A Suffused would've just figured something out that works. An Occultist would speak strange words compared to a mage as well.

We can agree on this.

**Comment by eunomicZenith** [comment]
I see now that I misread something and caused a lot of fuss for nothing, as mlenser never suggested letting suffuseds cast spells without verbal/somatic components. My apologies.

We can agree that the suffused shouldn't need foci or material components.
We can agree different classes should have different looking/sounding verbal/somatic components.

Personally, I think foci and non-expensive material components can be painlessly removed from the system entirely.

**Comment by itamarcu** [comment]
I also think removing irrelevant material components is good.

Arcane foci, though... I kinda like giving players the option to disarm wizards, steal foci from them, and generally be able to take the wizard's spellbook and staff/wand/etc and throw the wizard in jail and the wizard will be neutralized (or able to only do cantrips). Little differences like this between characters can be interesting and can lead to interesting moments.

As for people who use weapons - one option is to let them use specific weapons as foci, so if they're disarmed they still lose some spellcasting. Another option is to have a special feat or feature that lets these not have to hold a focus. Another option is to limit what kind of spells require foci, like I suggested earlier, so you'd need an arcane focus for classic dramatic wizard spells but won't need them for most or all gish spells.

**Comment by DalenWBrauner** [comment]
Big fan of this discussion. Only points I'd add:

> I mean, I think social spells are made to be used subtly? What is the use of Friends and Judge and illusions if others know you are casting a spell and influencing them?

Sure your _target_ has no idea they're being influenced, but what about others nearby? Can a politician's bodyguard tell he's being influenced when the spell cast? The fact that this is ambiguous/left to DMs kinda really bothers me- I'm excited to see this resolved.

> giving players the option to disarm wizards, steal foci from them... and the wizard will be neutralized (or able to only do cantrips)

I like this, at least conceptually. I don't think there's any harm in having all casters require a Focus of some kind that is trivial to obtain and use, no matter what kind of caster you are. Something that would almost never be relevant, unless players actively began exploring the idea of disarming someone of their focus (or someone is jailed). 
Casting at only half your mana limit would be my first stab at a potential penalty.

This is _also_ somewhere Psions would be unique- their focus is literally their minds. It would take very special circumstances to hamper mental spellcasting of that nature.

(I also just had a huge deja vu... I feel like a discussion like this might have been one of the earliest issues I participated in.)

**Comment by eunomicZenith** [comment]
> I don't think there's any harm in having all casters require a Focus of some kind that is trivial to obtain and use, no matter what kind of caster you are. 

There _is_ an issue, if you're not allowing weapons as foci, and if you are, we start to get into the territory of discussing what can be a focus and whether it needs to be held, if any old weapon can be used as a focus or only specific ones (which presumably puts a hole in the 'trivial to obtain' and makes players yell at you when their new and powerful sword happens not to be a spellcasting focus).

**Comment by Paulorpribeiro** [comment]
Too many issues being discussed and, again, I'm late for the party. I few takes I'd like to add to this:

If we are to implement some somatic and verbal components to some spells, instead of adding components to each of the 700+ spells is to make a standard for each theme. Mind would have none, illusions, fear and charm (maybe) would also have none, exceptions apply. All other themes would have a standard of both verbal and somatic
Then we could say that psions would not use any components at all. Suffused could forgo the verbal components, but not the somatics.

About the foci issue, I think all spellcaster (maybe except psions, we need to make sure they dont get too overpowered in this regard) should be limited to half their mana limit without their foci. I mean, it is a classic thing to disarm the caster of their focus (disarming the staff, cutting their finger with the ring, stealing their medalion) to put the breakes on their power. You disarm a warrior of this weapon he is n or useless (he SHOULD carry other weapons), but also not as deadly.

TLDR: introduce components on a theme basis instead of a spell by spell basis; put exception to some classes; reintroduce foci as an item important but not essential to spelcasting.

**Comment by Marcloure** [comment]
I still disagree that foci should be reintroduced, but alright

**Comment by DalenWBrauner** [comment]
I think a seperate issue should be created for discussing Spell Foci- it could be spoken to at length on its own.

> introduce components on a theme basis instead of a spell by spell basis

Initially I suspected this wouldn't work, as you'd have too many "one exception" spells that would need different components from the rest of their theme. But then I went through a few themes and couldn't find any good counterexamples! Maybe this _could_ work.

**Comment by Paulorpribeiro** [comment]
It's definately easier than going through every spell. We could make the changes to each theme and then open issues for specific spells that need tinkering

**Comment by eunomicZenith** [comment]
I understand the desire to avoid labor and granularity, but IMO we can't do without them in this context. Every alternative is a hack-around that's bound to have failure points.

Taking your suggestion, what if it turns out there's a spell in a theme that normally has vocal components that doesn't want to have vocal components? What about a spell that's in two themes, one with vocal and another without vocal?

I think similar failure points will be found in any generalized rule, and that we just gotta swallow adding components to the spells.

**Comment by Paulorpribeiro** [comment]
I agree. After all, spells are all different, with some sharing some traits and mechanics. Perhaps trying to make a general assumption for each theme is a good starting point instead of a shortcut.

**Comment by itamarcu** [comment]
I'd still like to make a few honest attempts at creating a general rule for how components work. Example suggestion:

* The caster needs either a vocal or a somatic component, their choice, when casting a bonus action or a reaction spell. Spells that require an action or longer to cast require both a somatic and a vocal component.

* (Alternative) The caster needs either component, their choice, when casting a spell below their mana limit. If they expend mana equal to their mana limit, they must use both components.

* The caster may use a free hand or a hand holding an arcane focus to do the gestures necessary for a somatic component.

* You may make a Stealth/Deception check to hide the fact you are casting a spell. If you fail the save, either you are noticed or the spell fails, decided by your GM.

* There is a feat that allows you to use somatic components while holding a weapon or a shield.

* There is a feat that grants you the ability to cast spells with one less component than you would otherwise need (not including expended material components, of course). You may take it twice to become the equivalent of a psion, able to cast spells while bound and gagged.

**Comment by eunomicZenith** [comment]
I'm still personally strongly against requiring spellcasting foci for reasons already mentioned.

>You may make a Stealth/Deception check to hide the fact you are casting a spell. If you fail the save, either you are noticed or the spell fails, decided by your GM.

Why this rule? What scenario do you have in mind where being noticed causes you to fail to cast your spell?

> There is a feat that allows you to use somatic components with a hand that is holding a weapon or a shield (perhaps as part of War Caster, or with an additional small benefit tacked onto it).

Personally I truly hated how mandatory War Caster was in 5e, and I really don't want another similarly "pick this or else" feat here. I almost suggested "at least give it for free to gishes" but then I remembered that doesn't even work because of how often full-caster clerics join the fray, not to mention in general feeling like the pointless and unfun kind of restriction that just eats a feat.

(For the nth time, I will repeat I'm in favor of dropping foci entirely. Yes, you're probably sick of me saying it. Yes, I'm going to keep saying it every time someone suggests stuff for arcane foci that's not 'get rid of them'.)

**Comment by Paulorpribeiro** [comment]
About the foci, it's about time we open a different issue about it, or take it to discord and make a long discussion about it. I'd leave my take on this: It should be a variant that could substitute one or both component for a caster that wields it.

About the components: I like itamarcu's proposal, although I think spells that require one of V or S components should be in the rules, not player choice. With those guidelines it's not difficult (although require some work) to fill all 700+ spells with components

* bonus action - V or S
* reaction - V or S
* one action or longer - V and S
* only costly material components, no need to change anything
* strike spell variants never require S components (so they can be cast with weapons in hand)

Exception1 - If you cast a spell lower than your spell limit and you can eschew either the S or M component. 
Exception2 - psionicists always disregard V and S
Special: Devise a feat that allows to eschew one or two non-material components when casting spells.

Thoughts?

**Comment by itamarcu** [comment]
> I think spells that require one of V or S components should be in the rules, not player choice. With those guidelines it's not difficult (although require some work) to fill all 700+ spells with components

I don't think we should bother with that - the main reason why I'd make such a general all-encompassing rule is to avoid having to edit 700 spells :p

So yes, every spellcasters will need to read that section of the rules once or twice (or not, if it never needs to come up), but the section of the rules will be phrased pretty much like how you described it.

* I want all bonus action attack spells to be castable with only a V component, because many gishes will be holding two weapons or a weapon and a shield or a two-handed weapon
* I want all bonus action teleportation-escape spells to be V, as this keeps the (popularly known) ability to teleport out of a bind
* After going through a lot of spells, it looks like almost all bonus action spells are either a weapon attack buff when hitting (e.g. Searing Strike), a long weapon buff before hitting (e.g. Shillelagh), a temporary defense (usually +2 to AC), teleportation, or a small utility (e.g. Grapnel). Most of these, I'd like to be Verbal, but can imagine some being Somatic.
* Reaction spells are mostly defenses, but also include important things such as Counterspell, Inspire, or Cutting Words. I think most of these make sense as Somatic, but a few (such as Cutting Words) should obviously be Verbal only.
* If we make all spells Verbal, Silence and Gag might be too strong. I'd like spellcasters to be able to cast at least *some* spells while silent.
* If we make any bonus/reaction spell Somatic, it hurts the viability of this spell by two-hands-holding-weapons gishes (unless they happen to be one of the exceptions, psions or having an extra feat). Every spell we add a mandatory somatic component to will have to be a spell that we want "dueling" gishes to use but not the more weapon-based gishes to use. I think this division is close enough to "actions" vs "reactions/bonus-actions".

If anyone can bring up good reasons why certain spells must be V or S or both or not both, I'd like that.

**Comment by DalenWBrauner** [comment]
> with only a V component, because many gishes will be holding two weapons or a weapon and a shield or a two-handed weapon

and

>If we make any spell Somatic, it hurts the viability of this spell by two-hands-holding-weapons gishes

Are both referencing the same problem: that Somatic components, by definition, require hand gestures,
which seem to be difficult when you're holding something.

What if we just allowed for somatic components to be made while holding objects like swords and shields? Let Paladins still direct a holy beam as long as they raise your sword to the heavens and then aim it at their foe? 
Then the way to deny somatic components would be if your hands are properly restricted from moving, or if you're carrying a heavy box that doesn't allow much for gesturing.

As long as the gestures are still obvious to onlookers, and requires you to still move your wrists and arms, the somatic component still serves its mechanical purpose, right?

**Comment by itamarcu** [comment]
That's a good point, actually. I guess the one thing that irks me about that solution is the fact that I'd still like spellcasters to need to either have a free hand or hold an arcane focus, when casting a big spell. Otherwise, when there is no mechanical reason to do so, all wizards will be dual-wielding daggers, and point their dagger at an enemy while conjuring a fireball, which just feels wrong to me.

Plus, I'd like to keep the "[one hand](https://qph.fs.quoracdn.net/main-qimg-6833a9084b5672d14168765d58e9e664) [holding a sword](https://cdna.artstation.com/p/assets/images/images/000/962/346/large/li-qian-1.jpg), [other hand holding](https://4.bp.blogspot.com/-Z4CKgIVb9eg/VhXOlO898RI/AAAAAAAADwM/eR4Rel0-sMw/s1600/75396.jpg) [a ball of magic](https://shutupinternet.files.wordpress.com/2014/08/gish.jpg)" trope fueled by mechanics to encourage it.

**Comment by DalenWBrauner** [comment]
> Otherwise, when there is no mechanical reason to do so, all wizards will be dual-wielding daggers,

All min-maxing wizards, perhaps. And you won't have this with too many weapons, as you still have to be proficient in them.

As much as I like the idea of incentivizing Wizards and Psions to keep their hands free for aesthetic purposes, we're moving into 'arcane focus' territory here, which should be discussed elsewhere.

Any other thoughts on this rule change for Somatic components? I suspect it clears a hurdle that keeps getting in the way of solving the original issue.

**Comment by Marcloure** [comment]
How is a Wizard wielding two daggers not ok? It's an aesthetic alright, and it isn't better than not using two daggers, since they are probably better with cantrips

**Comment by itamarcu** [comment]
> we're moving into 'arcane focus' territory here, which should be discussed elsewhere.

I don't think so - the arcane focus argument is "who should hold an arcane focus and who can have a free hand", IMO. I imagined both of these as "costing" one of your hands, and if you can do somatic components regardless of this, forcing only some of the casters to hold an arcane focus puts a more meaningful mechanical imbalance (since, e.g., perhaps this ends up with Battlemages having to hold focus-and-sword but Bloodrager allowed to dual wield, ending up just as effective in their spells but dealing more damage with their weapon attacks).

> How is a Wizard wielding two daggers not ok? It's an aesthetic alright, and it isn't better than not using two daggers, since they are probably better with cantrips

Well, just my personal opinion that it's a weird aesthetic. It's better than not using two daggers because your cantrips might be weaker if the enemy is resistant to magic, resistant to your damage type, or standing right next to you (if you got only ranged cantrips).

**Comment by Paulorpribeiro** [comment]
However you want (or not want) to deal with foci, I would consider a hand holding a focus as a a hand available to provide the somatic components.
Another question we need answer in order to move forward: should a two-handed gish be able to cast a spell as powerful as a sword and board gish? Or as a single weapon gish? In my opinion, the answer is no, and that is why I advocate for gish spells to require somatic components as well. 
On the other hand a paladin should be able to emblazon a holy symbol in their shields and magus should be able to use their weapons as focus (as per the weapon bond thing) and thus ignore the somatic component (a focus in the hand counts meets the somatic components requirement).
But if said magus grabs a magic glaive and does not have time to attune with it, or the paladin looses his shield, then they have to use their hand to cast.

**Comment by DalenWBrauner** [comment]
> should a two-handed gish be able to cast a spell as powerful as a sword and board gish? Or as a single weapon gish?

I would say yes. They can now, and I don't see value in changing that.

Spells are already limited when you're a gish. Further limiting what spells you can cast or your mana limit is depending on which weapons you're holding might be fun if you like to min-max your characters, or if you're playing a third-person video game, but is needlessly restrictive for players that just want to make a cool character who can cast spells _and_ swing a sword.

**Comment by mlenser** [comment]
I've locked the conversation on this thread as we're already at 50 replies.

This has brought about very strong opinions and I think its best if we either discuss via discord or focus on very specific parts of the conversation and resolve them one at a time.
